src/lando/api/tests/test_landings.py

```diff
@@ -283,26 +347,26 @@ def test_integrated_execute_job(
     ), "Successful landing should trigger Phab repo update."
 
 
+@pytest.mark.parametrize(
+    "repo_type",
+    [
+        SCM_TYPE_GIT,
+        SCM_TYPE_HG,
+    ],
+)
 @pytest.mark.django_db
 def test_integrated_execute_job_with_force_push(
-    hg_server,
-    hg_clone,
+    repo_mc,
     treestatusdouble,
     monkeypatch,
     create_patch_revision,
+    repo_type: str,
 ):
-    treestatusdouble.open_tree("mozilla-central")
-    repo = Repo.objects.create(
-        scm_type=SCM_TYPE_HG,
-        name="mozilla-central",
-        url=hg_server,
-        required_permission=SCM_LEVEL_3,
-        push_path=hg_server,
-        pull_path=hg_server,
-        force_push=True,
-        system_path=hg_clone.strpath,
-    )
+    repo = repo_mc(repo_type)
+    repo.force_push = True
```

Comment:
I think rather than manually overriding this here, it would be better to have a repo "factory" fixture that is used by all the repo fixtures above, and then you can just pass it additional overriding parameters if needed. That way, if we ever had any hooks or triggers that rely on either the database state of the repo, or things that happen upon repo creation, this type of override won't cause any problems.

This Repo object should be saved before proceeding, since the value is not going to be reflected in line 378 (when repos are fetched from the db).